src/main/java/com/aquarium/aquarium_backend/Controllers/MeasurementController.java

```diff
@@ -27,6 +28,12 @@ public ResponseEntity<List<Measurement>> getAllMeasurements() {
     return new ResponseEntity<>(allMeasurements, HttpStatus.OK);
   }
 
+  @GetMapping("userSensor/{userSensorId}")
+  public ResponseEntity<List<Measurement>> getMethodName(@PathVariable Long userSensorId) {
+    return new ResponseEntity<>(
+        measurementService.getMeasurementsByUserSensorId(userSensorId), HttpStatus.OK);
+  }
+
```

Comment:
getMethodName is not a good method name 🗡️

Reply:
Done